app/repositories/sift/ApplicationSiftRepository.scala

```diff
@@ -247,5 +247,26 @@ class ApplicationSiftMongoRepository(
       )
     })
   }
+
+  def findAllUsersInSiftEntered: Future[Seq[FixUserStuckInSiftEntered]] = {
+
+    val query = BSONDocument("applicationStatus" -> ApplicationStatus.SIFT,
```

Comment:
I think it's possible to skip SIFT_READY, e.g. if you have only PDFS (Form required: true, evaluation required: false) - so in that case this would still return true. I'd perhaps add a _SIFT_COMPLETE -> exists false_ as well